beefy-gadget/src/worker.rs

```diff
@@ -227,6 +238,8 @@ where
 				debug!(target: "beefy", "🥩 New Rounds for id: {:?}", active.id);
 
 				self.best_beefy_block = Some(*notification.header.number());
+
+				metric_set!(self, beefy_best_block, *notification.header.number());
```

Comment:
Depending on what we want the metric exactly to be we might or might not want to have this line. This is kind of a "forced BEEFY change", cause the block is not really beefy-finalized (as metric decsription suggests). I think it's fine for now, since we want to have insight about internal state, but would add a comment here to make sure we get rid of this later on.

Reply:
Yeah, getting rid of this one was the idea. Will clarify.